scripts/docker/build.gradle

```diff
@@ -0,0 +1,174 @@
+//Note: this section 'buildscript` is only for the dependencies of the buildscript itself.
+// See the second 'repositories' section below for the actual dependencies of GATK itself
+buildscript {
+    repositories {
+        mavenCentral()
+        jcenter() // for shadow plugin
+     }
+}
+
+
+plugins {
+    id "java"           // set up default java compile and test tasks
+    id 'maven'
```

Comment:
I doubt we need maven here

Reply:
i think we need it to download jacoco? we point to maven central still.

Reply:
The maven plugin is for releasing to maven, not downloading from it.  That's a core gradle thing I think.